app/src/main/java/io/pslab/communication/ScienceLab.java

```diff
@@ -2989,7 +3038,7 @@ public void loadEquation(String channel, String function) {
                     y.add(Math.sin(x.get(i)));
                     break;
                 case "tria":
-                    y.add(abs(x.get(i) % 4 - 2) - 1);
+                    y.add(Math.abs(x.get(i) % 4 - 2));
```

Comment:
Please add a comment explaining that the modulo operator in java returns the negative remainder for negative operands, unlike python's modulo operator which always returns the positive remainder.